src/Collections-Native/WordArray.class.st

```diff
@@ -22,12 +22,12 @@ WordArray >> atAllPut: value [
 	super atAllPut: value
 ]
 
-{ #category : #accessing }
+{ #category : #'objects from disk' }
```

Comment:
in ArrayedCollection you moved this method from 'objects from disk' to accessing

Reply:
Yes - this was done intentionally - because if you look at all the other implementors of #byteSize
it is in "accessing"

Reply:
well, then why you move WordArray>>#byteSize to "objects from disk"? It is like that in the image bootstrapped from this PR